sbi/utils/conditional_density_utils.py

```diff
@@ -294,12 +294,16 @@ def __init__(
         self.device = self.potential_fn.device
         self.allow_iid_x = allow_iid_x
 
-    def __call__(self, theta: Tensor, track_gradients: bool = True) -> Tensor:
+    def __call__(
+        self, theta: Tensor, x_o: Tensor, track_gradients: bool = True
+    ) -> Tensor:
```

Comment:
why did x_o get added to this?

Reply:
because the new check in `base_posterior` requires the passed `potential_fn` Callable to have both `theta` and `x_o` args. 
https://github.com/sbi-dev/sbi/blob/17f3033aa55514ac6c039da729ee84ce32eb20d9/sbi/inference/posteriors/base_posterior.py#L47-L54

Reply:
I will remove this change from this PR and make a separate issue / PR